osc-ui/src/main/resources/org/osc/core/broker/view/common/VmidcMessages.properties

```diff
@@ -216,5 +210,10 @@ sg.overlap.port = Member: ''{0}'' is already part of existing Security Group(s).
 is not already protected.
 sg.overlap.port.sgname = Member: ''{0}'' is already part of existing Security Group(s). Check that one of the ports of the entity \
 is not already protected as part of ''{1}''.
+sg.membership.caption = Member Information
+sg.no.members = No Active Security Members Present
 
-os.extension.not.present = Extension: ''{0}'' is not present in the OpenStack environment you are attempting to connect to. 
+os.extension.not.present = Extension: ''{0}'' is not present in the OpenStack environment you are attempting to connect to.
+
+general.ipaddr = IP Address
```

Comment:
IP Addresses?

Reply:
will use IP Address(es) because its 1 or more...not necessarily always more than 1